src/net/sourceforge/kolmafia/textui/parsetree/ProxyRecordValue.java

```diff
@@ -1754,39 +1753,29 @@ public String get_nickname() {
   }
 
   public static class ElementProxy extends ProxyRecordValue {
-    public static RecordType _type =
+    public static final RecordType _type =
         new RecordBuilder().add("image", DataTypes.STRING_TYPE).finish("element proxy");
 
     public ElementProxy(Value obj) {
       super(_type, obj);
     }
 
     public String get_image() {
-      switch ((Element) this.content) {
-        case NONE:
-          return "circle.gif";
-        case COLD:
-          return "snowflake.gif";
-        case HOT:
-          return "fire.gif";
-        case SLEAZE:
-          return "wink.gif";
-        case SPOOKY:
-          return "skull.gif";
-        case STENCH:
-          return "stench.gif";
+      return switch ((Element) this.content) {
           // No image for Slime or Supercold in Manuel
-        case SLIME:
-          return "circle.gif";
-        case SUPERCOLD:
-          return "circle.gif";
-      }
-      return "";
+        case NONE, SLIME, SUPERCOLD -> "circle.gif";
```

Comment:
It seems like this could be replaced with `Element.getColor()`, which would be doubly nice because it would remove duplicated information in the codebase. Depending on whether we care about an extremely minor breaking change, you might want to add `Element.getLegacyColor()` that forces SLIME and SUPERCOLD to return circle.gif instead of the images in the Element enum